web/src/components/team-member/team-member.css

```diff
@@ -17,35 +16,23 @@ img.team-member__photo, .team__accent-dot {
 }
 
 .alina-photo {
-  top: 200px;
+  top: 300px;
   left: 200px;
 }
 
 .ellie-photo {
-  top: 40px;
-  left: 240px;
+  top: 140px;
+  left: 140px;
 }
 
 .lian-photo {
-  bottom: 20px;
+  /* bottom: 20px; */
```

Comment:
is this going to be uncommented out later?

Reply:
that was an accident.  I just made a PR with what I had, but i didnt have time to polish it.